### DISCUSSION TITLE
Columns().AdjustToContents(1, 1) results in an font not found error

### DISCUSSION THREAD

**Discussion by StanislawFlexpedia** [ClosedXML/ClosedXML#2006] [Q&A]
I dont know if i should post it as a bug or im just missing something obvious, so im posting it here first.
Code runs perfectly fine but when i add:
`
ws.Columns().AdjustToContents(1, 1);
`
right before saving the file (still in using block) i get an error about missing fonts.
If i comment this line workbook is generated normally.

> System.ArgumentException: Unable to find font Calibri or fallback font Microsoft Sans Serif. Install missing fonts or specify a 
> different fallback font through 'LoadOptions.DefaultGraphicEngine = new DefaultGraphicEngine("Fallback font name")'. Additional 
> information is available at https://closedxml.readthedocs.io/en/latest/tips/missing-font.html page.

(i have this font installed)

**Comment by igitur** [comment]
Did you read the doc in that url?
